lib/replace_parameters.js

```diff
@@ -248,19 +238,17 @@ module.exports = class ParameterReplacer {
 
     _getEntityListKey(entityType) {
         switch (entityType) {
-        case 'tt:username':
-        case 'tt:contact':
-        case 'tt:email_address':
-        case 'tt:phone_number':
-            return ['string', 'tt:person_first_name'];
-
-        case 'tt:hashtag':
-            return ['string', 'tt:word'];
-        case 'tt:path_name':
-            return ['string', 'tt:path_name'];
-
-        default:
-            return ['entity', entityType];
+            case 'tt:username': case 'tt:contact': case 'tt:email_address':
+                return ['string', 'tt:person_first_name'];
+            case 'tt:phone_number':
+                return ['string', 'tt:phone_number'];
+            case 'tt:hashtag':
+                return ['string', 'tt:word'];
+            case 'tt:path_name':
+                return ['string', 'tt:path_name'];
+
+            default:
+                return ['entity', entityType];
```

Comment:
Please preserve the existing format of switch statements, thanks.

(The existing format is **not** WebStorm's preferred format, which looks like what you're doing here)

Reply:
lol  yeah  I did not intend to but WebStorm forced me on the indentations.
Is there a reason to not indent?

Reply:
It's how our style is defined. Style is pretty much arbitrary, but it needs to be consistent.